proposals/lip-0061.md

```diff
@@ -264,13 +285,17 @@ The functions returns `True` if and only if the aggregate certificate signature
 ###### Execution
 
 ```python
-verifyAggregateCertificateSignature(keysList, weights, threshold, networkIdentifier, c):
+def verifyAggregateCertificateSignature(validatorList: list[Validator], threshold: uint64, chainID: bytes, c: Certificate) -> bool:
```

Comment:
1. @ricott1 Could you incorporate it in the Interoperability updates?
2. Note that this is a stateless function that could be implemented as a library. I think in terms of specifications, it is fine to just reference the function definition. In the implementation it could mean that some code is duplicated or written as a library.

Reply:
yes.

Reply:
Fine for me (although I personally prefer when PRs are complete and do not break anything). I leave it open for @ricott1

Reply:
I will resolve this comment then.